hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -3053,4 +3069,12 @@ public boolean supportsSelectAliasInGroupByClause() {
 		return false;
 	}
 
+	public boolean supportsStoredProcedures() {
+		return true;
+	}
+
+	public boolean supportsComputedIndexes() {
+		return true;
+	}
+
```

Comment:
in my opinion it is better just to use the `@SkipForDialect(value = CockroachDB1920Dialect.class,...)` annotation and avoid all these new methods.